.github/workflows/concrete_python_tests_linux.yml

```diff
@@ -111,5 +111,6 @@ jobs:
             export COMPILER_BUILD_DIRECTORY=/build
             export KEY_CACHE_DIRECTORY=/tmp/KeySetCache
             make pytest
+            rm -rf /tmp/KeySetCache
```

Comment:
I am not sure the problem exists on linux, we don't have dedicated instances.

Reply:
It might exist in the future, better be safe than sorry.